torch/csrc/cuda/Graph.cpp

```diff
@@ -87,5 +87,30 @@ void THCPGraph_init(PyObject* module) {
           "debug_dump",
           torch::wrap_pybind_function_no_gil(
               &::at::cuda::CUDAGraph::debug_dump),
-          py::arg("debug_path"));
+          py::arg("debug_path"))
+      .def_static(
```

Comment:
TODO: Do we need to make wrappers for these new functions in https://github.com/pytorch/pytorch/blob/5b5a1f520207232ab6b2856c057946cf2786b75f/torch/cuda/graphs.py#L117 for the sake of the docs ?